e2e-tests/playwright/support/server/default_config.ts

```diff
@@ -72,6 +72,7 @@ const onPremServerConfig = (): Partial<TestAdminConfig> => {
             EnableOnboardingFlow: false,
             EnableSecurityFixAlert: false,
             GiphySdkKey: 's0glxvzVg9azvPipKxcPLpXV0q1x1fVP',
+            PostPriority: true,
```

Comment:
This can be reverted since the default config has this setting already - https://github.com/mattermost/mattermost/blob/master/e2e-tests/playwright/support/server/default_config.ts#L179